code/datums/manufacturing.dm

```diff
@@ -1108,6 +1108,15 @@ ABSTRACT_TYPE(/datum/manufacture)
 	create = 1
 	category = "Clothing"
 
+/datum/manufacture/speech_pro
```

Comment:
I'm going to say don't make it manufacturable for now because I feel like people are going to give them to mimes every round/expect mimes to use them.